Vision/Vision.bs

```diff
@@ -23,28 +23,15 @@ Status Text:
 Boilerplate: omit conformance
 </pre>
 
-# Intention # {#intention}
```

Comment:
"Intention" is an odd word, was introduced in the port to BS, and is not how I would have described this section.